dev-libs/libical/libical-2.1.0~.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="An implementation of iCalendar protocols and data formats"
```

Comment:
Yep, it is intentional, but @raefaldhia just forgot to rename the file to **`libical-2.1.0~git.recipe`** (which is, if I'm not mistaken, the convention to use when we know that 2.1.0 has not been released yet, and that the current git commit will be part of that release when it becomes available.)